content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md

```diff
@@ -0,0 +1,29 @@
+Title: Announcing the winner of "Kenneth Gonsalves Award 2016"-Dr.Ajith Kumar B.P!
```

Comment:
Remove quote. Make the title as "Winner of the 2016 Kenneth Gonsalves Award - Dr. Ajith Kumar B.P!"